packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-input.component.tsx

```diff
@@ -87,9 +120,8 @@ const VitalsBiometricInput: React.FC<VitalsBiometricInputProps> = ({
                     render={({ field: { onBlur, onChange, value, ref } }) => (
                       <NumberInput
                         allowEmpty
-                        className={`${styles.textInput} ${disabled && styles.disabledInput} ${val.className} ${
-                          !isWithinNormalRange && styles.danger
-                        }`}
+                        placeholder={generatePlaceholder(val.name)}
+                        className={`${styles.textInput} ${disabled && styles.disabledInput} ${val.className}`}
```

Comment:
This is slightly wordier, but doesn't result in unexpected behaviour (the `&&` version either sets the class name or adds the class `false`, which is not what we want.

```suggestion
                        className={`${styles.textInput} ${disabled ? styles.disabledInput : ''} ${val.className}`}
```